internal/services/cdn/cdn_frontdoor_rule_resource.go

```diff
@@ -678,12 +678,12 @@ func resourceCdnFrontDoorRuleRead(d *pluginsdk.ResourceData, meta interface{}) e
 	ctx, cancel := timeouts.ForRead(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	id, err := parse.FrontDoorRuleID(d.Id())
+	id, err := parse.FrontDoorRuleIDInsensitively(d.Id())
```

Comment:
this is coming from the Resource ID/is validated at import time so we can parse this case-sensitively:

```suggestion
	id, err := parse.FrontDoorRuleID(d.Id())
```

Reply:
Fixed.